madmom/io/__init__.py

```diff
@@ -0,0 +1,548 @@
+# encoding: utf-8
+"""
+Input/output package.
+
+"""
+
+from __future__ import absolute_import, division, print_function
+
+import io as _io
+import sys
+import contextlib
+
+import numpy as np
+
+from .audio import load_audio_file
+from .midi import load_midi, write_midi
+from ..utils import suppress_warnings, string_types
+
+if sys.version_info[0] < 3:
+    _file_handle = file
+else:
+    from io import TextIOBase as _file_handle
+
+
+ENCODING = 'utf8'
+
+# dtype for numpy structured arrays that contain labelled segments
+# 'label' needs to be castable to str
+SEGMENT_DTYPE = [('start', np.float), ('end', np.float), ('label', object)]
+
+
+def _open_file(fn):
```

Comment:
This function is probably unnecessary. It can be replaced with `open_file`, I guess.